### PR TITLE
chore/update dashboard dependencies

### PR DESCRIPTION
**Pull request by thismax**
- update cerberus-dashboard module to use `react-scripts`
- remove non-idiomatic/unsupported `@connect` and `@redux-form` decorators
- update gradle wrapper
- relocated files
- updated project formatting

### CODE REVIEW

**Review by fieldju — APPROVED** [review]
@thismax I didn't read every single file but the approach you outlined in the PR description of moving to the CRA scripts and updating the things it doesn't support (decorators) and applying a formatter sounds pretty solid.

**Comment by fieldju** [comment]
The build is failing @mayitbeegh because there are some new CVE's that need to be addressed.
That is unrelated to this PR, and is also causing @rdong333's PR build to fail

File: cerberus-dashboard/package.json [hunk 1]
```diff
@@ -43,31 +45,24 @@
...
-    "webpack": "1.13.1",
-    "webpack-dev-server": "1.14.1"
+    "react-scripts": "^3.4.0",
+    "redux-devtools": "3.3.1"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
```
Comment (mayitbeegh) [inline cerberus-dashboard/package.json:60]: What's with opera mini?

**Review by melanahammel — APPROVED** [review]

File: cerberus-dashboard/package.json (hunk 1 above)
Comment (thismax) [reply cerberus-dashboard/package.json:60]: this is the default browser list for `react-scripts`:  https://create-react-app.dev/docs/supported-browsers-features/

**Pull request merged**